CppCoreGuidelines.md

```diff
@@ -9365,73 +9364,75 @@ If you have a naked `new`, you probably need a naked `delete` somewhere, so you
 
 (Simple) Warn on any explicit use of `new` and `delete`. Suggest using `make_unique` instead.
 
-### <a name="Rr-immediate-alloc"></a>R.12: Immediately give the result of an explicit resource allocation to a manager object
+### <a name="Rr-immediate-alloc"></a>R.12: Instead of manual resource allocation, use factory functions that return RAII objects
 
 ##### Reason
 
 If you don't, an exception or a return may lead to a leak.
 
-##### Example, bad
+##### Example
 
-    void f(const string& name)
+    void bad(const char *name)
     {
-        FILE* f = fopen(name, "r");            // open the file
-        vector<char> buf(1024);
-        auto _ = finally([f] { fclose(f); });  // remember to close the file
+        FILE *f = fopen(name, "r");
+        std::vector<char> buf(1024);
         // ...
     }
 
-The allocation of `buf` may fail and leak the file handle.
-
-##### Example
+If `buf`'s constructor throws, then `FILE *f` will not be fclosed — a resource leak.
```

Comment:
How about `fclose`d

Reply:
travis-ci picks it up, too
```
Warning: Spellcheck failed, fix words or add to dictionary:
Line 9382: fclosed -> fclose
```

Reply:
Changed to "`fclose`d", although if that makes the spellchecker any happier, I'll be amazed.